### PR TITLE
DOP-4652 update skeleton for dark mode

### PR DESCRIPTION
**Pull request by seungpark**
### Stories/Links:

DOP-4652

### Current Behavior:

[Example of white loader in unrelated staging link](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/cloud-docs/maya/DOP-4638/reference/api-resources-spec/changelog/index.html) - you can select two different resource versions to see the loader, but dark theme does not have a dark skeleton loader

### Staging Links:

[Staging link with loaders mocked](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/cloud-docs/seung.park/DOP-4652/reference/api-resources-spec/changelog/index.html)

### Notes:
- Address [this comment](https://jira.mongodb.org/browse/DOP-4652?focusedCommentId=6452897&focusedId=6452897&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6452897) about missing max width on this page
- Uses LG/skeleton loader to show loading skeletons vs `react-loading-skeleton`

### README updates

- - [ ] This PR introduces changes that should be reflected in the README, and I have made those updates.
- - [X] This PR does not introduce changes that should be reflected in the README

### CODE REVIEW

File: src/templates/changelog.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import MainColumn from '../components/MainColumn';
 import { theme } from '../theme/docsTheme';
 
 const Wrapper = styled(MainColumn)`
-  max-width: unset;
+  max-width: 775px;
```
Comment (caesarbell) [inline src/templates/changelog.js:8]: @seungpark, I am asking only for learning purposes, where did we get the `775px` value from?

**Review by caesarbell — APPROVED** [review]
LGTM - overall looks good to me. I just posted one question for learning purposes.

File: src/templates/changelog.js (hunk 1 above)
Comment (mmeigs) [reply src/templates/changelog.js:8]: ^ same. It looks great, but the max-width is now extremely different. It feels squished even on a normal laptop screen.

Comment (seungpark) [reply src/templates/changelog.js:8]: this was included in the [Notes comment](https://jira.mongodb.org/browse/DOP-4652?focusedCommentId=6452897&focusedId=6452897&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6452897), but it seems like that linked comment was misunderstood. We want to center and space like the landing pages <[jira comment](https://jira.mongodb.org/browse/DOP-4652?focusedCommentId=6454582&focusedId=6454582&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6454582)>. Will update this

Comment (mmeigs) [reply src/templates/changelog.js:8]: Aha, sorry - didn't see that. You are so right! That space really makes me sad, but out of our hands!

**Review by mmeigs — APPROVED** [review]

File: src/templates/changelog.js (hunk 1 above)
Comment (seungpark) [reply src/templates/changelog.js]: this was removed in favor of grid columns similar to product landing!

**Pull request merged**